src/keyboard/Key.tsx

```diff
@@ -33,17 +34,31 @@ interface KeyDimension {
 
 function makeSize(
   { width, height }: KeyDimension,
-  oneU: number
+  oneU: number,
```

Comment:
In the future, if you want to do some general clean up *and* feature work, please at least split them into discrete commits in the PR. As is, this is harder to review since I can't quickly see what changes are *actually* part of the fix, and what are refactoring/style changes.

Reply:
absolutely, I've cleaned this PR up that removes extraneous diff.